externalized/game.json

```diff
@@ -55,6 +55,12 @@
     //  vanilla value: 0
     "enemy_weapon_minimal_status": 0,
 
+    //  Make the AI a bit smarter and let it target heads and legs sometimes
+    //  vanilla values: false, 0, 0
+    "mod_ai_a": false,
```

Comment:
Can we call this differently? Maybe `ai_targets_body_parts`? Or more generic (so we can add stuff later) `mod_ai_aiming`?